docs/aws.md

```diff
@@ -0,0 +1,76 @@
+# AWS Setup Instructions
+The goal of this document is to provide a guide on how to set up the GitHub Action runner on AWS. This document will guide you through the account setup required to use this action with AWS.
+
+## Prerequisites
+- An AWS account
+
+## Setup
+1. Prepare an IAM user with AWS access keys
+    1. Go to the AWS Management Console and sign in to your account.
+    2. Go to the IAM console.
+    3. In the navigation pane, choose Policies.
+    4. Create a new policy by clicking "Create Policy".
+    5. Next, click JSON and paste the following JSON with the required permissions and then click "Next".
```

Comment:
Is there an AWS Managed policy that matches this? That might simplify a few of these steps.

Reply:
The closest Managed Policy for this is `AmazonEC2FullAccess` which contains the following permission set:
```json
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Action": "ec2:*",
            "Effect": "Allow",
            "Resource": "*"
        },
        {
            "Effect": "Allow",
            "Action": "elasticloadbalancing:*",
            "Resource": "*"
        },
        {
            "Effect": "Allow",
            "Action": "cloudwatch:*",
            "Resource": "*"
        },
        {
            "Effect": "Allow",
            "Action": "autoscaling:*",
            "Resource": "*"
        },
        {
            "Effect": "Allow",
            "Action": "iam:CreateServiceLinkedRole",
            "Resource": "*",
            "Condition": {
                "StringEquals": {
                    "iam:AWSServiceName": [
                        "autoscaling.amazonaws.com",
                        "ec2scheduled.amazonaws.com",
                        "elasticloadbalancing.amazonaws.com",
                        "spot.amazonaws.com",
                        "spotfleet.amazonaws.com",
                        "transitgateway.amazonaws.com"
                    ]
                }
            }
        }
    ]
}
```